tools/proofers/preview.js

```diff
@@ -424,7 +425,10 @@ var makePreview = function (txt, viewMode, wrapMode, styler) {
                 if (postChar === "\n") {
                     reportIssue(start, tagLen, "nlAfterStart");
                 }
-                if (/\S/.test(preChar)) { // non-space before start tag
+
+                // disallow letter or ,.;: before inline start tag
+                let disallowBeforeStart = XRegExp("\\pL|[,.;:]", "A");
```

Comment:
Just making sure that I'm understanding correctly that letters and `,.;:` cannot precede an inline start tag, but numbers are okay? So, something like `6<i>s</i>` would be okay, but `i<i>magine</i>` would not?

Should `!?` be included in that prohibition?

Reply:
This change is mostly reverting the change I made for unicode with some other changes, one of which is to let numbers be ok in this siuation.
I think it is not really a good idea to change to different features at the same time so I am proposing to close this MR and instead make a new MR which just reverts the previous one. Then we can make any changes for unicode or other things later.